UM/Trust.py

```diff
@@ -451,9 +461,29 @@ def signedFolderPreStorageCheck(self, path: str) -> bool:
                     self._violation_handler(f"Manifest '{manifest_path}' is not properly self-signed in '{path}'.")
                     return False
 
-                # Otherwise, as far as this quick pre check is concerned, there is nothing wrong:
-                Logger.log("i", f"Central storage file signed correctly for '{path}'.")
-                return True
+            # Check if the central storage file doesn't contain files that would be moved outside the plugin folder,
+            #   or files that would be moved to outside of the central storage location:
```

Comment:
```suggestion
            #  or files that would be moved to outside of the central storage location:
```

Reply:
On purpose, to show that the sentence continues. Don't mind it lined out though, but then you'd want to remove both spaces, not just one.

Reply:
Fixed in https://github.com/Ultimaker/Uranium/pull/717/commits/81250f7af3c809cf213a6464016698bfeb9c0c12